### PR TITLE
Base Goerli cWETHv3 deploy

### PR DESCRIPTION
**Pull request by kevincheng96**
This PR introduces the cWETHv3 deployment on Base Goerli as well as the proposal to initialize the market there.

When initializing the cWETHv3 market, we bridge over some WETH reserves. However, Optimism/Base does not support bridging WETH directly, so we bridge over ETH instead and wrap it before sending it to cWETHv3. I had to add some logic to our mock relayer code to support bridging of ETH.

TODOs:

- [x] deploy Base WETH testnet instance
- [x] run market initialization proposal

### CODE REVIEW

File: deployments/base-goerli/weth/deploy.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    'pricefeeds/ConstantPriceFeed.sol',
+    [
+      8,                                             // decimals
+      exp(1, 8)                                      // constantPrice
+    ]
+  );
+
+  // Deploy scaling price feed for cbETH
+  // XXX there is no cbETH / ETH pricefeed on testnet
+  // const cbETHScalingPriceFeed = await deploymentManager.deploy(
```
Comment (scott-silver) [inline deployments/base-goerli/weth/deploy.ts:27]: did you intend to leave this commented-out price feed in?

File: deployments/base-goerli/weth/deploy.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  //   'cbETH:priceFeed',
+  //   'ScalingPriceFeed.sol',
+  //   [
+  //     '0xcD2A119bD1F7DF95d706DE6F2057fDD45A0503E2', // cbETH / ETH price feed
+  //     8                                             // decimals
+  //   ]
+  // );
+  const cbETHScalingPriceFeed = await deploymentManager.deploy(
```
Comment (scott-silver) [inline deployments/base-goerli/weth/deploy.ts:35]: `cbETHConstantPriceFeed`?

File: scenario/constraints/ProposalConstraint.ts [hunk 3]
```diff
@@ -75,7 +75,13 @@ export class ProposalConstraint<T extends CometContext> implements StaticConstra
             await executeOpenProposal(governanceDeploymentManager, proposal);
           }
           debug(`${label} Open proposal ${proposal.id} was executed`);
+        }
+        catch (err) {
```
Comment (scott-silver) [inline scenario/constraints/ProposalConstraint.ts:79]: super nit, but maybe put the `catch` on the same line as the close bracket:

```
} catch (err) {
```

**Review by scott-silver — APPROVED** [review]
💪

File: deployments/base-goerli/weth/deploy.ts (hunk 1 above)
Comment (kevincheng96) [reply deployments/base-goerli/weth/deploy.ts:27]: Yeah I did this on purpose just so when I work on the mainnet deploy script, I'll remember that it'll be different from the testnet one.

File: scenario/constraints/ProposalConstraint.ts (hunk 3 above)
Comment (kevincheng96) [reply scenario/constraints/ProposalConstraint.ts:79]: Yeah that's more preferable for sure.

File: deployments/base-goerli/weth/deploy.ts (hunk 2 above)
Comment (kevincheng96) [reply deployments/base-goerli/weth/deploy.ts:35]: Good catch, done.

**Comment by kevincheng96** [comment]
Contracts deployed via Github Actions: https://github.com/compound-finance/comet/actions/runs/5179574342/jobs/9332634761

**Comment by kevincheng96** [comment]
Proposal created via Github Actions: https://github.com/compound-finance/comet/actions/runs/5182218070/jobs/9338742372

Proposal details: https://compound.finance/governance/proposals/77?target_network=goerli

Proposal execution on Base Goerli: https://goerli.basescan.org/tx/0x15f6445e1b2b5f2cab8b0a2577c32203d076e45dd47fadb0ac7ee38c27a8d577

**Pull request merged**